app/models/email_settings.rb

```diff
@@ -9,7 +9,9 @@ class EmailSettings
                 :service_email_body,
                 :service_email_pdf_heading,
                 :service_email_pdf_subheading,
-                :service_csv_output
+                :service_csv_output,
+                :send_by_email_dev,
+                :send_by_email_production
```

Comment:
if we are using environment specific attrs now do we still need `send_by_email`?

Reply:
So this is the main thing I wasn't sure about.  I added these accessors purely so that I could do `:"send_by_email_#{env}"` without it goving me errors...

From what I can see the controller does some adjustments to send the right value through...  I wasn;t sure if there was some reason for things being done that way, so I didn't touch toon much.

We could adjust it if you'd prefer though?